stdlib/string.gr

```diff
@@ -2184,7 +2184,8 @@ let trimString = (stringPtr: WasmI32, byteLength: WasmI32, fromEnd: Bool) => {
     // TODO(#661): Use unicode whitespace property and unicode line terminator
     if (!fromEnd) {
       if (
-        byte == 0xEFn && // Check for the first BOM byte
+        byte == 0xEFn
+        && // Check for the first BOM byte
```

Comment:
This comment being on the line below really feels like it changes the meaning of it.  There has been a few cases like this.